bin/yii

```diff
@@ -41,6 +43,15 @@ use Yiisoft\Yii\Console\Application;
 
     /** @var ContainerInterface $container */
     $container = $params['console']['container']();
+    $eventConfigurator = $container->get(EventConfigurator::class);
+    $eventConfigurator->registerListeners(require Builder::path('events-console'));
 
-    $container->get(Application::class)->run();
+    $application = $container->get(Application::class);
+    try {
+        $application->start();
+        $exitCode = $application->run(null, new ConsoleBufferedOutput());
```

Comment:
What's the purpose of ConsoleBufferedOutput?

Reply:
It is for debugger